### DISCUSSION TITLE
FileNotFoundException: Could not load file or assembly 'System.Runtime.InteropServices.RuntimeInformation, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

### DISCUSSION THREAD

**Discussion by Workaholic01** [dlemstra/Magick.NET#1654] [General]
Hi Everyone,

I am getting this exception while instantiating MagickImage
```
using (MagickImage image = new MagickImage(inputMemoeryStream))
       {
           // Write the image to the output stream in AVIF format
           image.Write(outputStream, MagickFormat.Avif);

           // Optionally, write the output stream to a file
           return outputStream.ToArray();
       }
   }
```

**Project Properties**
![image](https://github.com/dlemstra/Magick.NET/assets/29353695/b8ecef04-773f-48fd-a93a-79bc520ca2de)

**Tech stack:**
- .Net Framework 4.8
- Magick.NET-Q8-AnyCPU
- Magic.NET.Core

I tried adding this System.Runtime.InteropServices.RuntimeInformation Package but it still giving exception even if I install 4.0.0.0. Anything I am missing?

**Comment by dlemstra** [comment]
Can you create a tiny project that can be used to reproduce this issue?

**Reply by Workaholic01** [reply]
I just tried in the sample console project with same version / settings. It is working fine with that but when I am trying it in my web app it throws exception.

System.TypeInitializationException: 'The type initializer for 'NativeMagickSettings' threw an exception.'

**Reply by dlemstra** [reply]
I suspect this must be caused by something in the setup of your web project. Maybe some old binding redirects?

**Reply by Workaholic01** [reply]
Yes, that is correct. It was due to some unnecessary bindings / dependent assemblies in web.config. Removing them did the job. Works fine now. Thank you. :)

**Reply by dlemstra** [reply]
Thanks for coming back here and responding so other people can find your solution in the future.
